### PR TITLE
better admin view

### PR DESCRIPTION
**Pull request by RClift**
basic formatting for the admin page, needs some formatting fixes (second column buttons) and coloring scheme.  things can get moved around after general formatting is set.  functionality is next large step including pulling list of users and information from a database into a table

### CODE REVIEW

File: src/pages/Admin.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+              <IonCol size="2">Team</IonCol>
+
+              <IonCol size="2">Email</IonCol>
+
+              <IonCol size="2">Total Steps</IonCol>
+
+              <IonCol size="3">Actions</IonCol>
+            </IonRow>
+
+            <IonRow>
```
Comment (bailee-crandley) [inline src/pages/Admin.tsx:59]: Would it be possible to turn this into more of a looping scenario? Like create a list of all of your fake data objects and then say something like, "for every object in list, display another row"? I think we're going to need it when we start pulling data from an actual database. You should check out Nathan's Manual Step Recording PR for an example of using functions in views.

**Comment by bailee-crandley** [comment]
Could you please add screenshots of this page to a comment in this PR?

**Comment by bailee-crandley** [comment]
Richard is going to review comment and add screenshots, then team will review.

**Review by NathanMoes — APPROVED** [review]

**Comment by RClift** [comment]
added some more fixes in, more to come, will try to get some screenshots or a video in when i can

**Pull request closed without merging**